Xamarin.Forms.Platform.MacOS/Renderers/RadioButtonRenderer.cs

```diff
@@ -44,6 +50,11 @@ protected override void OnElementChanged(ElementChangedEventArgs<RadioButton> e)
 					btn.SetButtonType(NSButtonType.Radio);
 					SetNativeControl(btn);
 					ObserveStateChange(true);
+
+					btn.Activated += HandleActivated;
+
+					if (e.NewElement.IsChecked)
```

Comment:
@chucker please review :)

Reply:
Interestingly, it seems like this change is _not_ necessary once you merge the renderer into its regular class. It seems only needed if there's a separate subclass, as in https://github.com/hyvart/Unjammit/pull/48

You can verify this as follows: Xamarin.Forms.ControlGallery.MacOS, RadioButton Gallery, RadioButton Group (Attached Property, Binding).

Modify its XAML (`Xamarin.Forms/Xamarin.Forms.Controls/GalleryPages/RadioButtonGalleries/RadioButtonGroupBindingGallery.xaml`) to have one option be checked already:

```xml
                <RadioButton Content="Option A" Value="A" Grid.Row="2"></RadioButton>
                <RadioButton Content="Option B" Value="B" Grid.Row="2" Grid.Column="1"></RadioButton>
                <RadioButton Content="Option C" Value="C" Grid.Row="3" IsChecked="True"></RadioButton>
                <RadioButton Content="Option D" Value="D" Grid.Row="3" Grid.Column="1"></RadioButton>
```

With your custom subclass, my addition is needed for Option C to be checked. Merging it back together into one class, that doesn't seem necessary (nor effectual; the code never seems to be run).

Reply:
Makes sense! There are some behaviors that can't be overridden by sub-classes because they exist in private members.

I'll revert that last change, then.